content/guides/getting-started/devs-vs-users.md

```diff
@@ -13,34 +12,32 @@ when used in conjunction with other components?).
 ### Developer Concerns
 
 Developers care about the API of the component. They want to make sure that if
-they exercise the component in a particular way, with certain props, that it'll
+they exercise the component in a particular way, with certain props, it'll
 behave the way they expect it to. A test written for a developer provides
 coverage over the component's usage and signature.
 
-Component testing written with developers in mind heavily covers props, events
-(Vue), and children/slots (React + Vue).
+Component testing written with developers in mind heavily covers props, events, and children.
```

Comment:
This change is a suggestion to make the sentence framework-agnostic. Events and children are generic terms for any framework